File: beginner/python/1866.py
n = int(input())

for i in range(0,n):

    soma = 0

    v = int(input())

    for j in range(1,v+1):
        if(j % 2 != 0):
            soma += 1
        else:
            soma -= 1
        
    print(soma)

File: beginner/javascript/1866.js
var input = require('fs').readFileSync('stdin', 'utf8').split('\n');

var i, j, n, cont = 1, v, soma;

n = parseInt(input.shift());

for(i = 0; i < n; i++){

    soma = 0;

    v = parseInt(input.shift());

    for(j = 1; j <= v; j++){
        if(j % 2 != 0){
            soma += cont;
        } else{
            soma -= cont;
        }
    }

    console.log(soma);
}